CMakeLists.txt

```diff
@@ -34,9 +34,38 @@ endif()
 include (CTest)
 include (CheckIncludeFiles)
 
+#do not add or build any tests of the dependencies
+set(original_run_e2e_tests ${run_e2e_tests})
+set(original_run_int_tests ${run_int_tests})
+set(original_run_unittests ${run_unittests})
+
+set(run_e2e_tests OFF)
+set(run_int_tests OFF)
+set(run_unittests OFF)
+
+if(NOT ${use_installed_dependencies})
+    if ((NOT TARGET azure_macro_utils_c) AND (EXISTS ${CMAKE_CURRENT_LIST_DIR}/deps/azure-macro-utils-c/CMakeLists.txt))
+        add_subdirectory(deps/azure-macro-utils-c)
+        include_directories(deps/azure-macro-utils-c/inc)
+    endif()
+
+    if(${original_run_int_tests} OR ${original_run_unittests})
```

Comment:
>${original_run_int_tests} OR ${original_run_unittests} [](start = 7, length = 54)

ARe you missing original_run_e2e_tests here.  I know that this doesn't have e2e test, but would it be useful to have?

Reply:
It would only matter when umock receives e2e tests. But in all honesty umock would never get such tests (also there's a bit of a blur between e2e/int tests at this level;).

---
In reply to: [270218328](https://github.com/Azure/umock-c/pull/59#discussion_r270218328) [](ancestors = 270218328)